### PR TITLE
[backend/frontend] Add missing sortable

### PR DESCRIPTION
**Pull request by RomuDeuxfois**
### Proposed changes

* Fix sortable possibilities

### CODE REVIEW

File: openbas-front/src/admin/components/components/teams/Teams.tsx [hunk 1]
```diff
@@ -94,8 +96,8 @@ const Teams = () => {
   const headers = [
     { field: 'team_name', label: 'Name', isSortable: true },
     { field: 'team_description', label: 'Description', isSortable: true },
```
Comment (savacano28) [inline openbas-front/src/admin/components/components/teams/Teams.tsx:98]: sortable: false?

Comment (RomuDeuxfois) [reply openbas-front/src/admin/components/components/teams/Teams.tsx:98]: I add this on back-end
@Queryable(sortable = true)
    @Column(name = "team_description")
    @JsonProperty("team_description")
    private String description;

Comment (savacano28) [reply openbas-front/src/admin/components/components/teams/Teams.tsx:98]: sorry, I said for team_name

Comment (RomuDeuxfois) [reply openbas-front/src/admin/components/components/teams/Teams.tsx:98]: Oh I rollback your changes.. sorry

Comment (RomuDeuxfois) [reply openbas-front/src/admin/components/components/teams/Teams.tsx:98]: Fix

**Review by savacano28 — APPROVED** [review]

**Pull request merged**